consensus/src/consensus/inner/commit.rs

```diff
@@ -14,6 +14,8 @@
 // You should have received a copy of the GNU General Public License
 // along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
 
+use std::time::Instant;
```

Comment:
```suggestion
```

Reply:
`Instant` is used here.

Reply:
it is, but via `std::time::Instant::now()`, so either this import is unnecessary (and it's the CI that complained about it), or its use in line 153 needs to be adjusted to avoid the warning